plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/DeltaLakeSchemaSupport.java

```diff
@@ -549,6 +597,11 @@ public static Set<String> unsupportedReaderFeatures(Set<String> features)
         return Sets.difference(features, SUPPORTED_READER_FEATURES);
     }
 
+    public static Set<String> unsupportedWriterFeatures(Set<String> features)
+    {
+        return Sets.difference(features, SUPPORTED_WRITER_FEATURES);
```

Comment:
A static method works now, but what if we support for a new writer feature is conditional based on config or session?

I think it would be safer to have this as a instance method in DeltaLakeMetadata.
I see however this is same as in `unsupportedReaderFeatures`, so it's already committed state. Fine to keep for now.